assets/app/lib/settings.rb

```diff
@@ -6,8 +6,25 @@
 module Lib
   module Settings
     DARK = `window.matchMedia('(prefers-color-scheme: dark)').matches`.freeze
-    # http://mkweb.bcgsc.ca/colorblind/ 12 color palette
-    ROUTE_COLORS = %i[#A40122 #008DF9 #00FCCF #FF5AAF #8400CD #FF6E3A #FFB2FD #FFC33B].freeze
+    # http://mkweb.bcgsc.ca/colorblind/ 15 color palette, with some substitutions + 1 additional
+    ROUTE_COLORS = %i[
+      #A40122
+      #099FFA
+      #00DCB5
+      #FF5AAF
+      #9400E6
+      #FF6E3A
+      #009581
+      #FFDC3D
+      #EF0096
+      #7CFFFA
+      #005FCC
+      #F60239
+      #00E307
+      #FFCFE2
+      #AFFF2A
+      #E8D2AB
+    ].freeze
```

Comment:
hyperbole ;-) 2 lines would suffice.

Reply:
It was a lot easier to reorder this way.

Reply:
OK, fixed.